### ISSUE TITLE
Initializing a New Repo in .\src\VBAProject or .\src\Filename.ext

### ISSUE THREAD

**Issue by eteichm** [rubberduck-vba/Rubberduck#2106]
I tested Source-control as described in [Getting Started](https://github.com/rubberduck-vba/Rubberduck/wiki/Source-Control:-Getting-Started)

When Initializing a New Repository via menu Manage, RD creates the new repository in the directory chosen by the user (and not in the Default repository location) in a folder named as the name of the VBAProject. 

I suggest to create the new repo by default under directory `src\` in the same directory as the file itself in a folder named as the name of the VBAProject or as the filename.  
In other words, I suggest the Default repository location be `.\src\VBAProject` or `.\src\FileName.ext` 
E.g: if the file FileName.ext is located in C:\Users{username}\Documents\MyXLSMfiles\
then the new repo is created by default in C:\Users{username}\Documents\MyXLSMfiles\src\VBAProject or C:\Users{username}\Documents\MyXLSMfiles\src\FileName.ext 

Naming the folder as the name of the file seems to be a better option. 
This suggestion is what I'm actually practicing with [vbaDeveloper](https://github.com/hilkoc/vbaDeveloper)

**Comment by rubberduck203** [comment]
Thanks for suggesting this. 
Unfortunately, we decided to remove the experimental feature. 

Please see #3758
